pom.xml

```diff
@@ -900,6 +901,15 @@
                                             </sourcePoms>
                                             <destinationPom>${basedir}/integration-tests/azure-grouped/pom.xml</destinationPom>
                                         </mergePom>
+                                        <mergePom>
+                                            <sourcePoms>
+                                                <directory>${basedir}/integration-test-groups/compression</directory>
+                                                <includes>
+                                                    <include>*/pom.xml</include>
+                                                </includes>
+                                            </sourcePoms>
+                                            <destinationPom>${basedir}/integration-tests/compression-grouped/pom.xml</destinationPom>
+                                        </mergePom>
```

Comment:
By similarity, I've also added this. I wonder if it's really useful as I was able to execute the compression grouped tests without this `mergePom` previously ?